frameworklauncher/src/main/java/com/microsoft/frameworklauncher/applicationmaster/ApplicationMaster.java

```diff
@@ -1447,4 +1471,16 @@ public boolean existsLocalVersionFrameworkRequest() throws NotAvailableException
   protected ClusterConfiguration getClusterConfiguration() {
     return requestManager.getClusterConfiguration();
   }
+
+  protected Configuration getConfiguration() {
+    return this.conf;
+  }
```

Comment:
Init it into SelectionManager instead of add here.

Reply:
Do not use it like am.requestManager, etc. They are designed to be private, and changed to protected only for Testing

---
In reply to: [174688127](https://github.com/Microsoft/pai/pull/318#discussion_r174688127) [](ancestors = 174688127)